### PR TITLE
feat: add trail scene for custom material testing

### PR DESCRIPTION
**Pull request by wumaolinmaoan**
## Summary by CodeRabbit

- **New Features**
  - Implemented a new trail effect with a dedicated fragment shader.
  - Enhanced texture management with inheritance from `EffectsObject`.

- **Enhancements**
  - Improved blending property settings for materials.
  - Optimized constants usage in post-processing passes for better performance and readability.
  - Updated texture creation functions for better integration with the Three.js library.

- **Refactor**
  - Reordered and modified import statements for better project structure.
  - Updated class constructors to include new parameters for consistency across packages.
  - Refactored various methods for improved texture handling and material management.

- **Style**
  - Updated CSS styling for the demo container to adjust the layout.

- **Documentation**
  - Added import statement documentation for clarity in usage.

### CODE REVIEW

File: packages/effects-core/src/components/effect-component.ts [hunk 1]
```diff
@@ -45,20 +46,53 @@ export class EffectComponent extends RendererComponent implements Disposable {
           type: glContext.FLOAT,
           size: 3,
           data: new Float32Array([
-            -1, 1, 0, //左上
-            -1, -1, 0, //左下
-            1, 1, 0, //右上
-            1, -1, 0, //右下
+            -24.1306, 0, 80.0432, 25.2414, 0, 80.0413, 24.7642, 0, 76.9534, -23.6524, 0, 76.9554, -19.4331, 0, 157.237, 20.542, 0, 157.237, 21.329, 0, 154.149, -20.2199, 0, 154.149, 22.0945, 0, 151.062, -20.985, 0, 151.062,
```
Comment (yiiqii) [inline packages/effects-core/src/components/effect-component.ts:49]: 这些数据是临时的还是就这样固定了？

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/components/effect-component.ts (hunk 1 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/components/effect-component.ts:49]: 临时的，后面会调整为从 json 读取资产数据

Comment (SuRuiMeng) [reply packages/effects-core/src/components/effect-component.ts:49]: 加个TODO吧

File: packages/effects-core/src/ticker.ts [hunk 2]
```diff
@@ -25,7 +25,7 @@ export class Ticker {
   }
   setFPS (fps: number) {
     this.targetFPS = clamp(fps, 1, 120);
-    this.interval = Math.floor(1000 / fps) - 1;
+    this.interval = Math.floor(1000 / fps) - 2;
```
Comment (yiiqii) [inline packages/effects-core/src/ticker.ts:28]: 这个改动应该提到日常迭代，尽快上了

**Review by yiiqii — CHANGES_REQUESTED** [review]

File: packages/effects-core/src/ticker.ts (hunk 2 above)
Comment (wumaolinmaoan) [reply packages/effects-core/src/ticker.ts:28]: 好的

**Review by RGCHN — APPROVED** [review]

**Review by SuRuiMeng — APPROVED** [review]

**Review by yiiqii — APPROVED** [review]

**Pull request merged**